package.json

```diff
@@ -25,11 +25,17 @@
     "async"
   ],
   "devDependencies": {
-    "async": "~0.9.0",
-    "jasmine-focused": "~1.0.7"
+    "async": "~2.6.2",
+    "grunt": "^1.0.4",
```

Comment:
Why did you add the `grunt` dependencies?

Reply:
Because the readme says `Run grunt to compile the CoffeeScript and native code` and apparently you need to install grunt and other related dependencies before running grunt. So I added them to devDependencies for future developers. No idea why they were not added earlier.